e2e-tests/pitr/conf/pitr-job.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: batch/v1
```

Comment:
why this file needed?

Reply:
removed https://github.com/percona/percona-xtradb-cluster-operator/pull/697/commits/867e25afaa236becf1253b02c8bf67e3c397219e